### PR TITLE
translate howto/chart_repository_sync_example.md

### PR DESCRIPTION
**Pull request by sunminjeon**
from #627
Signed-off-by: sunmin.jeon@samsung.com
Translate howto/chart_repository_sync_example in Korean

https://helm.sh/docs/howto/chart_repository_sync_example/

### CODE REVIEW

**Comment by sunminjeon** [comment]
@singlemocha @jmnote @bluefriday
리뷰 부탁드립니다.

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+## 전제 조건
+* [gsutil](https://cloud.google.com/storage/docs/gsutil) 툴을 설치해야 한다. *gsutill rsync 기능이 필요하다.*
+* 헬름 바이너리에 대한 접근 권한이 있어야 한다.
+* _선택 사항: 실수로 파일을 삭제할 경우를 대비하여 GCS 버킷에 [오브젝트 버전 관리](https://cloud.google.com/storage/docs/gsutil/addlhelp/ObjectVersioningandConcurrencyControl#top_of_page)를 설정하는 것을 추천한다._
+
+## 로컬 차트 리포지토리 디렉토리 설정
+
+[차트 리포지토리 가이드]({{< ref
+"/docs/topics/chart_repository.md" >}})에서 실행했던 것처럼 로컬 디렉토리를 만들고, 패키지된 차트를 그 위치로 옮긴다.
```
Comment (bluefriday) [inline content/ko/docs/howto/chart_repository_sync_example.md:18]:
```suggestion
## 로컬 차트 리포지토리 디렉토리 설정
[차트 리포지토리 가이드]({{< ref
"/docs/topics/chart_repository.md" >}})에서 실행했던 것처럼 로컬 디렉토리를 만들고, 패키지된 차트를 그 위치로 옮긴다.
```

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+예제:
+```console
+$ mkdir fantastic-charts
+$ mv alpine-0.1.0.tgz fantastic-charts/
+```
+
+## 업데이트 된 index.yaml 생성
+`helm repo index` 헬름 명령어에 원격 리포지토리의 디렉토리 경로와 URL을 전달하여 업데이트 된 index.yaml 파일을 생성한다.
```
Comment (bluefriday) [inline content/ko/docs/howto/chart_repository_sync_example.md:27]:
```suggestion
`helm repo index` 헬름 명령어에 원격 리포지토리의 디렉토리 경로와 URL을 전달하여 
업데이트 된 index.yaml 파일을 생성한다.
```

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+```console
+$ helm repo index fantastic-charts/ --url https://fantastic-charts.storage.googleapis.com
+```
+그러면 업데이트 된 index.yaml 파일이 생성되고 `fantastic-charts/` 디렉토리에 위치하게 된다.
```
Comment (bluefriday) [inline content/ko/docs/howto/chart_repository_sync_example.md:32]:
```suggestion
그러면 업데이트 된 index.yaml 파일이 생성되고 
`fantastic-charts/` 디렉토리에 위치하게 된다.
```

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+## 로컬 및 원격 차트 리포지토리 동기화
+`scripts/sync-repo.sh` 명령어에 로컬 디렉토리명과 GCS 버킷명을 전달하여 디렉토리 컨텐츠를 GCS 버킷에 업로드한다.
```
Comment (bluefriday) [inline content/ko/docs/howto/chart_repository_sync_example.md:35]:
```suggestion
`scripts/sync-repo.sh` 명령어에 로컬 디렉토리명과 
GCS 버킷명을 전달하여 디렉토리 컨텐츠를 GCS 버킷에 
업로드한다.
```

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+Building synchronization state...
+Starting synchronization
+Copying file://fantastic-charts/alpine-0.1.0.tgz [Content-Type=application/x-tar]...
+Uploading   gs://fantastic-charts/alpine-0.1.0.tgz:              740 B/740 B
+Copying file://fantastic-charts/index.yaml [Content-Type=application/octet-stream]...
+Uploading   gs://fantastic-charts/index.yaml:                    347 B/347 B
+Congratulations your remote chart repository now matches the contents of fantastic-charts/
+```
+## 차트 리포지토리 업데이트
+차트 리포지토리 컨텐츠의 로컬 복사본을 보관하거나 `gsutil rsync`를 사용하여 원격 차트 리파지토리 컨텐츠를 로컬 디렉토리에 복사할 수 있다.
```
Comment (bluefriday) [inline content/ko/docs/howto/chart_repository_sync_example.md:59]:
```suggestion
차트 리포지토리 컨텐츠의 로컬 복사본을 보관하거나 `gsutil rsync` 를 사용하여 
원격 차트 리파지토리 컨텐츠를 로컬 디렉토리에 
복사할 수 있다.
```

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+Copying gs://bucket-name/alpine-0.1.0.tgz...
+Downloading file://local-dir/alpine-0.1.0.tgz:                        740 B/740 B
+Copying gs://bucket-name/index.yaml...
+Downloading file://local-dir/index.yaml:                              346 B/346 B
+```
+
+유용한 링크:
+* [gsutil rsync](https://cloud.google.com/storage/docs/gsutil/commands/rsync#description) 에 대한 문서
+* [차트 리포지토리 가이드]({{< ref "/docs/topics/chart_repository.md" >}})
+* Google Cloud Storage의 [오브젝트 버전 관리 및 동시성 제어](https://cloud.google.com/storage/docs/gsutil/addlhelp/ObjectVersioningandConcurrencyControl#overview)에 대한 문서
```
Comment (bluefriday) [inline content/ko/docs/howto/chart_repository_sync_example.md:81]:
```suggestion
* [gsutil rsync](https://cloud.google.com/storage/docs/gsutil/commands/rsync#description) 에 
  대한 문서
* [차트 리포지토리 가이드]({{< ref "/docs/topics/chart_repository.md" >}})
* Google Cloud Storage의 [오브젝트 버전 관리 및 동시성 제어]
  (https://cloud.google.com/storage/docs/gsutil/addlhelp/ObjectVersioningandConcurrencyControl#overview)에 
  대한 문서
```

**Review by bluefriday — COMMENTED** [review]
수고하셨습니다. 주로 유지보수를 위해 라인 수를 맞추는 등의 리뷰 내용 추가하였습니다.

**Comment by bridgetkromhout** [comment]
I think the build is broken because this file is missing from the `ko` translation: `docs/topics/chart_repository.md`

10:43:28 PM: ERROR 2020/09/16 03:43:28 [ko] REF_NOT_FOUND: Ref "/docs/topics/chart_repository.md": "/opt/build/repo/content/ko/docs/howto/chart_repository_sync_example.md:17:36": page not found
10:43:28 PM: ERROR 2020/09/16 03:43:28 [ko] REF_NOT_FOUND: Ref "/docs/topics/chart_repository.md": "/opt/build/repo/content/ko/docs/howto/chart_repository_sync_example.md:80:38": page not found
10:43:28 PM: Total in 3936 ms

**Comment by bluefriday** [comment]
> I think the build is broken because this file is missing from the `ko` translation: `docs/topics/chart_repository.md`
> 
> 10:43:28 PM: ERROR 2020/09/16 03:43:28 [ko] REF_NOT_FOUND: Ref "/docs/topics/chart_repository.md": "/opt/build/repo/content/ko/docs/howto/chart_repository_sync_example.md:17:36": page not found
> 10:43:28 PM: ERROR 2020/09/16 03:43:28 [ko] REF_NOT_FOUND: Ref "/docs/topics/chart_repository.md": "/opt/build/repo/content/ko/docs/howto/chart_repository_sync_example.md:80:38": page not found
> 10:43:28 PM: Total in 3936 ms

@bridgetkromhout 
yes, If #824 is merged, build error will be solved.

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+weight: 2
+aliases: ["/docs/chart_repository_sync_example/"]
+---
+
+*참고: 이곳의 예제들은 차트 리포지토리를 제공하는 Google Cloud Storage (GCS) 버킷에 맞게 작성되었다.*
+
+## 전제 조건
+* [gsutil](https://cloud.google.com/storage/docs/gsutil) 툴을 설치해야 한다. *gsutill rsync 기능이 필요하다.*
+* 헬름 바이너리에 대한 접근 권한이 있어야 한다.
+* _선택 사항: 실수로 파일을 삭제할 경우를 대비하여 GCS 버킷에 [오브젝트 버전 관리](https://cloud.google.com/storage/docs/gsutil/addlhelp/ObjectVersioningandConcurrencyControl#top_of_page)를 설정하는 것을 추천한다._
```
[the remark this answers is not in the document]
Comment (sunminjeon) [reply content/ko/docs/howto/chart_repository_sync_example.md:14]: gsutill 뒤의 링크는 숨김처리되기 때문에, 따로 줄처리를 하지 않았습니다.

File: content/ko/docs/howto/chart_repository_sync_example.md (hunk 6 above)
Comment (sunminjeon) [reply content/ko/docs/howto/chart_repository_sync_example.md:90]: gsutil rsync 뒤의 링크는 숨김처리 되기 때문에, 가독성을 위하여 줄처리를 하지 않았습니다.

**Comment by sunminjeon** [comment]
@bluefriday  리뷰 반영하였습니다. 확인 부탁드립니다. 감사합니다.

**Review by sunminjeon — COMMENTED** [review]
Signed-off-by: sunminjeon <sunmin.jeon@samsung.com>

File: content/ko/docs/helm/helm_completion.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+      --kube-token string                인증에 사용될 문자열 전달 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   이 값이 비어있지 않을 경우, 이 값에 지정된 디렉토리에 로그 파일 쓰기 수행
+      --log-file string                  이 값이 비어있지 않을 경우, 이 값에 지정된 파일명으로 로그 파일 쓰기 수행
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가 바이트이며 값이 0일 경우 최대 파일크기 제한 없음(기본 값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 : true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_completion.md:37]: Please correct this line: https://github.com/helm/helm-www/blame/master/content/en/docs/helm/helm_completion.md#L38

File: content/ko/docs/helm/helm_completion.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+## helm completion
+
+지정된 쉘에 대한 자동 완성 스크립트 생성
+
+### 개요
+
+
+지정된 쉘에 대한 자동 완성 스크립트를 생성한다.
+
+### Options
```
Comment (singlemocha) [inline content/ko/docs/helm/helm_completion.md:14]: 옵션

File: content/ko/docs/helm/helm_completion.md (hunk 8 above)
Comment (singlemocha) [reply content/ko/docs/helm/helm_completion.md:37]: correct translation should be below:
캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/.cache/helm/repository")

File: content/ko/docs/helm/helm_completion.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Helm Completion"
```
[the remark this answers is not in the document]
Comment (sunminjeon) [reply content/ko/docs/helm/helm_completion.md:2]: content/ko/docs/helm/ 경로의 타 한글 문서와 포맷을 맞추기 위해, title은 영문 그대로 (Helm Completion) 번역하는 게 자연스러울 듯 합니다.

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 11]
```diff
@@ -0,0 +1,90 @@
+---
+title: "차트 리포지토리 동기화"
+description: "로컬 및 원격 차트 리포지토리를 동기화하는 방법"
+weight: 2
+---
+
+*참고: 이곳의 예제들은 차트 리포지토리를 제공하는*
+
+*Google Cloud Storage (GCS) 버킷에 맞게 작성되었다.*
```
Comment (jmnote) [inline content/ko/docs/howto/chart_repository_sync_example.md:9]:
```suggestion
*참고: 이곳의 예제들은 차트 리포지토리를 제공하는
Google Cloud Storage (GCS) 버킷에 맞게 작성되었다.*
```

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+예제:
+```console
+$ mkdir fantastic-charts
+$ mv alpine-0.1.0.tgz fantastic-charts/
+```
+
+## 업데이트 된 index.yaml 생성
+`helm repo index` 헬름 명령어에 원격 리포지토리의 디렉토리 경로와 URL을 전달하여 
+
+업데이트 된 index.yaml 파일을 생성한다.
```
Comment (jmnote) [inline content/ko/docs/howto/chart_repository_sync_example.md:29]:
```suggestion
## 업데이트된 index.yaml 생성
`helm repo index` 명령어에 원격 리포지토리의 디렉토리 경로와 URL을 전달하여 
업데이트된 index.yaml 파일을 생성한다.
```

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+
+```console
+$ helm repo index fantastic-charts/ --url https://fantastic-charts.storage.googleapis.com
+```
+그러면 업데이트 된 index.yaml 파일이 생성되고 
+
+`fantastic-charts/` 디렉토리에 위치하게 된다.
```
Comment (jmnote) [inline content/ko/docs/howto/chart_repository_sync_example.md:36]:
```suggestion
그러면 업데이트된 index.yaml 파일이 생성되고 
`fantastic-charts/` 디렉토리에 위치하게 된다.
```

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 14]
```diff
@@ -0,0 +1,90 @@
...
+
+## 로컬 및 원격 차트 리포지토리 동기화
+`scripts/sync-repo.sh` 명령어에 로컬 디렉토리명과 
+
+GCS 버킷명을 전달하여 디렉토리 컨텐츠를 GCS 버킷에 업로드한다.
```
Comment (jmnote) [inline content/ko/docs/howto/chart_repository_sync_example.md:41]:
```suggestion
## 로컬 및 원격 차트 리포지토리 동기화
`scripts/sync-repo.sh` 명령어에 로컬 디렉토리명과 
GCS 버킷명을 전달하여 디렉토리 컨텐츠를 GCS 버킷에
업로드한다.
```

File: content/ko/docs/howto/chart_repository_sync_example.md [hunk 15]
```diff
@@ -0,0 +1,90 @@
...
+Copying file://fantastic-charts/alpine-0.1.0.tgz [Content-Type=application/x-tar]...
+Uploading   gs://fantastic-charts/alpine-0.1.0.tgz:              740 B/740 B
+Copying file://fantastic-charts/index.yaml [Content-Type=application/octet-stream]...
+Uploading   gs://fantastic-charts/index.yaml:                    347 B/347 B
+Congratulations your remote chart repository now matches the contents of fantastic-charts/
+```
+## 차트 리포지토리 업데이트
+차트 리포지토리 컨텐츠의 로컬 복사본을 보관하거나 `gsutil rsync` 를 사용하여
+
+원격 차트 리파지토리 컨텐츠를 로컬 디렉토리에 복사할 수 있다.
```
Comment (jmnote) [inline content/ko/docs/howto/chart_repository_sync_example.md:67]:
```suggestion
## 차트 리포지토리 업데이트
차트 리포지토리 컨텐츠의 로컬 복사본을 보관하거나 `gsutil rsync` 를 사용하여
원격 차트 리파지토리 컨텐츠를 로컬 디렉토리에 복사할 수 있다.
```

**Review by jmnote — COMMENTED** [review]
수고하셨습니다. 약간 수정해봤습니다. (영어판과 줄맞춤 등 포함)

**Comment by jmnote** [comment]
lgtm

**Review by bridgetkromhout — APPROVED** [review]
Thanks!

**Comment by bridgetkromhout** [comment]
Hello - some commits need their signoff fixed - please see https://github.com/helm/helm-www/pull/826/checks?check_run_id=1175951211 for details on that. 

Also, we have some build errors:

5:19:29 AM: ERROR 2020/09/28 10:19:29 [ko] REF_NOT_FOUND: Ref "/docs/topics/chart_repository.md": "/opt/build/repo/content/ko/docs/howto/chart_repository_sync_example.md:17:36": page not found
5:19:29 AM: ERROR 2020/09/28 10:19:29 [ko] REF_NOT_FOUND: Ref "/docs/topics/chart_repository.md": "/opt/build/repo/content/ko/docs/howto/chart_repository_sync_example.md:88:38": page not found

**Comment by sunminjeon** [comment]
I create new PR. 
https://github.com/helm/helm-www/pull/826

I will close this PR.

**Pull request closed without merging**